web/src/routes/EventsPage.tsx

```diff
@@ -87,15 +87,15 @@ const EventsPage = ({ city, pathname, languageCode, cityCode }: CityRouteProps):
     ),
   }
 
+  let returnedContent: React.ReactNode = null
```

Comment:
:x: Why did you change this file? Please revert

Reply:
I'm trying to reduce complexly 😓

Reply:
Hmmm, how are you trying to do this here? As far as I can see, you just changed `return` to `if/else` with an additional variable and fragment. Or am I missing something here?

Reply:
it says to many returns

Reply:
Code climate? I think in this case I'd ignore it. Also, this is not something you implemented/changed. I was just referring to new code. Sorry for the misunderstanding here.

Reply:
Yeah no worries..👍